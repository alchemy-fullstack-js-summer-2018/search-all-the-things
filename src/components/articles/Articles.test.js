import React from 'react';
import { shallow } from 'enzyme';
import Articles from './Articles';
import toJSON from 'enzyme-to-json';

const articles = [
  { id: 1 }, 
  { id: 2 }, 
  { id: 3 }
];

describe('Articles', () => {

  it('renders as designed', () => {
    const wrapper = shallow(<Articles articles={articles}/>);
    expect(toJSON(wrapper)).toMatchSnapshot();
  });

});