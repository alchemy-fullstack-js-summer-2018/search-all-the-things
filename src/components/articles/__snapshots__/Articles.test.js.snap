// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Articles renders as designed 1`] = `
<ul>
  <Article
    article={
      Object {
        "id": 1,
      }
    }
    key="0"
  />
  <Article
    article={
      Object {
        "id": 2,
      }
    }
    key="1"
  />
  <Article
    article={
      Object {
        "id": 3,
      }
    }
    key="2"
  />
</ul>
`;
